Microsoft.Dynamics365.UIAutomation.Browser/Logs/StringFormater.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
```

Comment:
This two classes can be useful also in project, to easily create more expressive Traces.